src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryDate.cs

```diff
@@ -561,12 +566,32 @@ private static bool TryGetCulture(string name, out CultureInfo value)
             }
         }
 
+        public static bool TryDateTimeParse(FormattingInfo formatInfo, IRContext irContext, StringValue value, out DateTimeValue result)
```

Comment:
>TryDateTimeParse [](http://example.com/codeflow?start=27&length=16)

this would change as per latest fetch...

Reply:
fetch main brn.